qos/evm/store.go

```diff
@@ -0,0 +1,96 @@
+package evm
+
+import (
+	"errors"
+	"math/rand"
+	"sync"
+
+	"github.com/buildwithgrove/path/relayer"
+)
+
+// EndpointStore provides the endpoint selection capability required
+// by the relayer package for handling a service request.
+var _ relayer.EndpointSelector = &EndpointStore{}
+
+// EndpointStoreConfig captures the modifiable settings
+// of the EndpointStore.
```

Comment:
I feel like it's easier to read this way and still short.  Please update elsewhere


```suggestion
// EndpointStoreConfig captures the modifiable settings of the EndpointStore.
```